README.md

```diff
@@ -18,32 +18,32 @@ Ignition Go provides a set of features to help with web server development. It i
 
```

Comment:
Github doesn't allow me to comment on things that were not changed in a PR, but the intro before this line here is using Ignition in a number of places.

Reply:
We can probably change documentation in a future PR, wdyt? We can probably add a task to follow up on this.

Reply:
That's okay with me.

Reply:
Task added to asana: https://app.asana.com/0/851925973517080/1203590389490573